### PR TITLE
DB-10999 fix maximumdisplaywidth if maxWidth has been used (show indexes)

### PR DESCRIPTION
**Pull request by martinrupp**

### CODE REVIEW

**Comment by martinrupp** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by spliceci** [comment]
TEST FAILED
http://nexus.splicemachine.com:8080/job/spliceengine-PR/job/Splice-PR-Branch/2358/

**Comment by spliceci** [comment]
TEST SUCCEEDED +1

File: db-tools-ij/src/main/java/com/splicemachine/db/impl/tools/ij/ijResultSetResult.java [hunk 1]
```diff
@@ -109,7 +109,7 @@ public ijResultSetResult(ResultSet r, ColumnParameters[] columnParameters) throw
 			int max = JDBCDisplayUtil.getMaxDisplayWidth();
 			if( max != 256 )
 			{
-				if (maxColWidth != 0 && (max == 0 || max > maxColWidth))
+				if (maxColWidth != 0 && (max != 0 && max > maxColWidth))
```
Comment (hatyo) [inline db-tools-ij/src/main/java/com/splicemachine/db/impl/tools/ij/ijResultSetResult.java:112]: I think `max != 0` can be removed.

Comment (hatyo) [reply db-tools-ij/src/main/java/com/splicemachine/db/impl/tools/ij/ijResultSetResult.java:112]: could you explain what the difference between `maxColWidth` and `max`?

**Review by hatyo — COMMENTED** [review]
Good work @martinrupp!

**Review by dgomezferro — APPROVED** [review]

File: db-tools-ij/src/main/java/com/splicemachine/db/impl/tools/ij/ijResultSetResult.java (hunk 1 above)
Comment (ascend1) [reply db-tools-ij/src/main/java/com/splicemachine/db/impl/tools/ij/ijResultSetResult.java:112]: Adding some comments to document the purpose behind the condition would be great.

**Review by ascend1 — APPROVED** [review]

File: db-tools-ij/src/main/java/com/splicemachine/db/impl/tools/ij/ijResultSetResult.java (hunk 1 above)
Comment (martinrupp) [reply db-tools-ij/src/main/java/com/splicemachine/db/impl/tools/ij/ijResultSetResult.java]: good point, done

**Comment by martinrupp** [comment]
addressed @hatyo and @ascend1 comments

**Review by carolp-503 — APPROVED** [review]
+1

**Pull request merged**